packages/cli-kit/src/store.ts

```diff
@@ -103,3 +104,18 @@ export function setThemeStore(store: string): void {
   debug(content`Setting theme store...`)
   cliKit.set('themeStore', store)
 }
+
+export function getSessionStore(): string | undefined {
```

Comment:
Should we drop `store` from the name of the new functions in this module? (i.e. `getSession`, `setSession`, and `removeSession`). It's a bit misleading seeing `store` in the name when the module already represents a `store`.

Reply:
Agreed!